src/x/time/time.go

```diff
@@ -31,7 +31,7 @@ const (
 
 // Now returns the current local time.
```

Comment:
is this comment still correct? doesn't seem to be local time anymore

Reply:
Oops yeah the comment is correct, should have reverted this change

Reply:
I think the comment is not correct, either. There is no notion of time zone on unix timestamp.
This `ToUnixNano` uses go `time.UnixNano()` under the hood, and see the end of it's comment:
```go
// UnixNano returns t as a Unix time, the number of nanoseconds elapsed
// since January 1, 1970 UTC. The result is undefined if the Unix time
// in nanoseconds cannot be represented by an int64 (a date before the year
// 1678 or after 2262). Note that this means the result of calling UnixNano
// on the zero Time is undefined. The result does not depend on the
// location associated with t.
func (t Time) UnixNano() int64 {
	return (t.unixSec())*1e9 + int64(t.nsec())
}
```
So I would change our comment to something more formally correct like "// Now returns the current Unix time."